### PR TITLE
Adds some jclouds Docker examples that work with getcarina.com

### PR DESCRIPTION
**Pull request by zack-shoylev**

### CODE REVIEW

**Comment by zack-shoylev** [comment]
The compute example does not work because of https://github.com/jclouds/jclouds-labs/pull/248

File: rackspace/pom.xml [hunk 1]
```diff
@@ -41,6 +41,12 @@
       <artifactId>jclouds-sshj</artifactId>
       <version>${jclouds.version}</version>
     </dependency>
+    <dependency>
+       <groupId>org.apache.jclouds.labs</groupId>
+       <artifactId>docker</artifactId>
+       <!-- TODO: Update version to variable jclouds.version on next release -->
```
Comment (nacx) [inline rackspace/pom.xml]: Can't this be done now?

Comment (zack-shoylev) [inline rackspace/pom.xml]: 1.9.2 seems broken compared to snapshot. I forgot what the problem was though, let me check

Comment (zack-shoylev) [inline rackspace/pom.xml]: Ok, I remembered. It was because of the privileged(false) that is needed in CreateComputeContainer

**Comment by zack-shoylev** [comment]
merged

File: rackspace/pom.xml (hunk 1 above)
Comment (nacx) [inline rackspace/pom.xml]: Cool thx! :)

**Pull request closed without merging**